src/config/sidebar.ts

```diff
@@ -886,8 +886,18 @@ export const SIDEBAR: Partial<Record<Sections, SectionEntry[]>> = {
           url: "ccip/tutorials/programmable-token-transfers-defensive",
         },
         {
-          title: "Transfer Tokens between EOAs",
-          url: "ccip/tutorials/cross-chain-tokens-from-eoa",
+          title: "Offchain",
+          url: "ccip/tutorials",
```

Comment:
Create a short off-chain landing page explaining why it is separate from the other tools. `ccip/tutorials/index.mdx` applies to the whole list of guides I think.